src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonApiMetricsConfig.java

```diff
@@ -0,0 +1,32 @@
+package io.stargate.sgv2.jsonapi.api.v1.metrics;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+@ConfigMapping(prefix = "stargate.metrics")
```

Comment:
maybe just `"stargate.jsonapi.metric` and all props directly in the top-level class

Reply:
Also `CONFIGURATION.md` should be updated..